src/docs/style/illustrations.mdx

```diff
@@ -0,0 +1,73 @@
+import { MaterialDesignDescription, ImageGrid } from '../../app/components';
+import { PictureAsPdf as PDFIcon } from '@material-ui/icons';
+import LightThemeImage from '../../app/assets/design-patterns/illustrations/illustration-light-theme.svg';
+import DarkThemeImage from '../../app/assets/design-patterns/illustrations/illustration-dark-theme.svg';
+import BackgroundShapeImage from '../../app/assets/design-patterns/illustrations/illustration-background.png';
+
+<!-- KEYWORDS: design illustration graph onboarding -->
+
+# Illustrations
+
+Illustrations are design elements that should be used mostly for onboarding screens and [empty states](/patterns/empty-states). Do not use illustrations where you are supposed to use icons.
+
+When creating your own illustrations, you should follow our guidelines as explained below. Alternatively, you may also choose to acquire existing artworks from repositories like [Icons8](https://icons8.com/illustrations), [Adobe Stock](https://stock.adobe.com/vectors), or [Humaaans Figma plugin](https://www.figma.com/community/plugin/739503328703046360). We recommend you finalizing the illustration with [Adobe Illustrator](https://www.adobe.com/products/illustrator.html), as it provides more export options.
+
+<React.Fragment>
+    <MaterialDesignDescription
+        title={'Onboarding'}
+        url={'https://material.io/design/communication/onboarding.html'}
+        description={`Material Design's guidelines on onboarding screens.`}
+    />
+    <MaterialDesignDescription
+        title={'Google Marketing Platform Visual Guidelines'}
+        url={
+            'https://designguidelines.withgoogle.com/ads-branding/assets/1hquKP2EcBlDaWyaLkoezqd8r9KPjRgnp/visual-guidelines-google-marketing-platform.pdf#'
+        }
+        description={`You are encouraged to follow their illustration guidelines.`}
+        avatar={<PDFIcon style={{ width: 40, height: 40 }} />}
+    />
+</React.Fragment>
+
+## Designing the Illustration
+
+### Perspective
+
+Your illustration should be either in a horizontal perspective (neither looking up nor looking down) or in parallel projection.
+
+### Ratio
+
+The width-height ratio should be between 1:1 to 2:3, which gives a margin on a mobile phone for UI interactions. If designing specifically for desktop applications (that is, the illustration will never show up on mobile), you can go wider than that, but no more than 16:9.
+
+### Drawing Style
+
+Keep your style consistent within one application, including your illustrations for the webpage for the product. Do not be too abstract or artistic, as your main goal is to inform your users of certain features in your product, instead of decorating it.
+
+### Text
+
+Avoid the use of text, as they might require translations, and can be hard to see on mobile. If you have to, use Open Sans, just like other PX Blue applications, and ensure that they are legit on the smallest targeted device.
```

Comment:
```suggestion
Avoid the use of text wherever possible. It can be hard to see on mobile and will be ineffective if your application needs to support multiple languages. If you have to use text, use Open Sans as the font and ensure that words are legible on your smallest targeted device.
```